src/site/_collections/post-to-collections.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
```

Comment:
I think this collection is only used by the SignPosts shortcode. Could it be removed and could the Signposts shortcode use the post-to-path filter to look up the slug it was given?

Reply:
Thinking about this a bit more, it might be good if we could give the SignPosts shortcode the title and href of the paths it needs to link to, instead of having it do the look up logic itself. My thinking is this might make it much more testable and reusable.

Reply:
Indeed, I did the same for ArticleNavigation lately. I think it is better to have separate "logic" shortcoder/macros and "UI" shortcodes just for visual output.